ome_zarr/napari.py

```diff
@@ -62,6 +62,24 @@ def f(*args: Any, **kwargs: Any) -> List[LayerData]:
                     layer_type = "labels"
                     if "colormap" in metadata:
                         del metadata["colormap"]
+                    if "properties" in metadata:
+                        props = metadata["properties"]
+                        reader_props = {}
+                        label_indices = list(props.keys())
+                        reader_props["index"] = label_indices
+
+                        # properties may be ragged, so we need to know all possible properties
+                        all_keys = set()
+                        for index in label_indices:
+                            all_keys = all_keys.union(set(props[index].keys()))
+
+                        # napari expects lists of equal length so we must fill with None
+                        for prop_key in all_keys:
+                            reader_props[prop_key] = [
+                                props[i][prop_key] if prop_key in props[i] else "None"
```

Comment:
This can probably be an actual `None` value here instead of a string `"None"` (unless it *has* to be a string for some reason). Since if some code is testing for the value being `None`, then the string won't be recognised as `None`.